test/binaryen.js/exception-handling.js

```diff
@@ -1,7 +1,8 @@
 function cleanInfo(info) {
   var ret = {};
   for (var x in info) {
-    if (x == 'id' || x == 'type' || x == 'name' || x == 'event') {
+    if (x == 'id' || x == 'type' || x == 'name' || x == 'event' ||
+        x == 'depth' || x == 'hasCatchAll') {
```

Comment:
Is this filtering out all expression children because their values are not useful? It would be good to have a comment here.

Reply:
Other values are basically address pointers, which can vary by execution. Other test cases within binaryen.js directory have this function too. (I added a few more fields here). Added a comment.